### DISCUSSION TITLE
강아지 장난감 추천

### DISCUSSION THREAD

**Discussion by yellow-jam** [SHAPE-UP/Bangida-client#40] [Polls]
더 좋은 의견은 댓글로 남겨주세요! 인형 모양도 받습니다!
저번에 유튜브에서 붕어빵모양 인형을 봤는데 정말 귀엽더라구요.

**Comment by otcroz** [comment]
여기 강아지 장난감 추천 링크입니다~!
강아지 장난감 고르실 때 참고하면 좋을 것 같아요
https://mypetlife.co.kr/113302/
